src/helpers/channels/update_stage_instance.ts

```diff
@@ -0,0 +1,41 @@
+import { rest } from "../../rest/rest.ts";
+import { Errors } from "../../types/discordeno/errors.ts";
+import { StageInstance } from "../../types/channels/stage_instance.ts";
```

Comment:
Use import type to import interfaces
```suggestion
import type { StageInstance } from "../../types/channels/stage_instance.ts";
```